client/src/modules/fiscal/fiscal.manage.html

```diff
@@ -41,7 +41,7 @@
             <select class="form-control"
               ng-model="FiscalManageCtrl.fiscal.previous_fiscal_year_id"
               ng-options="fy.id as fy.hrLabel for fy in FiscalManageCtrl.previous_fiscal_year">
-              <option value="" disabled>--{{ 'FORM.SELECT.FISCAL_YEAR' | translate }}--</option>
+              <option value="" disabled>--<span translate>FORM.SELECT.FISCAL_YEAR</span>--</option>
```

Comment:
Unfortunately, this will not work.  The HTML `<option>` element doesn't let you render HTML within it (that is part of the reason libraries like `ui-select` exist).  The correct way to implement this will be:
```html
 <option value="" disabled translate>FORM.SELECT.FISCAL_YEAR</option>
```